recipes/csa/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: csa
+    version: 1.22.0
+
+source:
+    git_url: https://github.com/sakov/csa-c.git
+    git_tag: v1.22.0
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    run:
+        - libgcc
```

Comment:
I am not 100% sure either. Adding `libgcc` worked out for me in `nco` and `cdo` to solve problems like [this one](https://github.com/ioos/conda-recipes/issues/723).